apps/test/unit/templates/manageStudents/manageStudentsReduxTest.js

```diff
@@ -118,6 +119,15 @@ const expectedBlankRow = {
 describe('manageStudentsRedux', () => {
   const initialState = manageStudents(undefined, {});
 
+  describe('toggleSharingColumn', () => {
```

Comment:
Looks good thanks for adding. Not necessary for this PR, since you aren't exposing the functionality, but when you do, can we have separate tests for showing the column and hiding the column.